client/my-sites/github-deployments/controller.tsx

```diff
@@ -113,6 +116,18 @@ export const redirectHomeIfIneligible: Callback = ( context, next ) => {
 		return;
 	}
 
+	const canManageOptions = canCurrentUser( state, siteId, 'manage_options' );
+
+	if ( ! canManageOptions ) {
+		context.store.dispatch(
```

Comment:
I'm wondering if we even need to show them a message. We just redirect to home as done in many other places without a message?